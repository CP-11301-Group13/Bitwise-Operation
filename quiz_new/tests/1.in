100 1
65438 65357 65495 65350 65445 65436 65454 65396 65359 65432 65314 65334 65505 65430 65354 65436 65433 65513 65340 65440 65306 65468 65368 65496 65437 65520 65448 65323 65291 65441 65322 65465 65515 65462 65526 65476 65478 65461 65481 65377 65498 65385 65291 65390 65431 65298 65451 65337 65503 65434 65362 65419 65458 65371 65346 65384 65287 65433 65439 65362 65521 65333 65432 65489 65323 65301 65444 65531 65511 65441 65477 65345 65285 65337 65299 65521 65519 65455 65417 65353 65380 65402 65485 65463 65281 65329 65410 65311 65295 65399 65303 65335 65397 65280 65418 65383 65402 65424 65462 65285