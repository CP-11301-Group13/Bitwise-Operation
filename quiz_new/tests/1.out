25014 42492 62084 26038 10994 44329 7508 13063 29085 5538 35639 52543 58614 42738 14899 17770 60396 45210 17231 26000 26140 49306 39349 49557 49998 8312 49482 27562 29653 14750 44209 56101 29279 22303 13086 14577 3139 28810 37687 12982 60087 32724 29584 19398 19949 52711 47105 64816 22865 5004 63610 43524 10786 12650